projects.yaml

```diff
@@ -57,6 +57,10 @@ projects:
         - *module_bl
         - *module_hic_sam3u2c
         - records/board/sam3u2c_bl.yaml
+    sam3u2c_mkit_dk_dongle_nrf5x_if:
+        - *module_if
+        - *module_hic_sam3u2c
+        - records/board/mkit_dk_dongle_nrf5x.yaml
     sam3u2c_nrf51mkit_if:
```

Comment:
The targets sam3u2c_nrf51mkit_if, sam3u2c_nrf51dk_if, sam3u2c_nrf51dongle_if can be removed since sam3u2c_mkit_dk_dongle_nrf5x_if takes their place